### PR TITLE
:sparkles: Added a button for external services to the About screen.

### LINKED ISSUE #183: Jump to DroidKaigi's twitter profile from the about screen
**Idea Description**
Jump to [DroidKaigi's twitter profile page](https://twitter.com/DroidKaigi) with tapped twitter icon in the about screen

blocked by #181 

**Reference images and links**

### LINKED ISSUE #184: Jump to DroidKaigi's medium from the about screen
**Idea Description**
Jump to [DroidKaigi's medium page](https://medium.com/droidkaigi) with tapped medium icon in the about screen

blocked by #181 

**Reference images and links**

### LINKED ISSUE #185: Jump to DroidKaigi's Youtube from the about screen
**Idea Description**
Jump to [DroidKaigi's Youtube page](https://www.youtube.com/c/DroidKaigi) with tapped youtube icon in the about screen

blocked by #181 

**Reference images and links**

### PR DESCRIPTION
**Pull request by Corvus400**
## Issue
- close #183 
- close #184 
- close #185 

## Overview (Required)
- As the title says.
  - Added icons for each external service.
  - Tapping the icon of each external service will take you to the respective service.
  - Supported cases where the application for each external service is not installed.
    - Added ChromeCustomTabs library.

## Links
- https://www.figma.com/file/NcSMs6dMsD88d4wOY0g3rK/DroidKaigi-2022-Conference-App?node-id=400%3A2942

## Screenshot
Before | After
:--: | :--:
|

### CODE REVIEW

File: feature-about/src/main/java/io/github/droidkaigi/confsched2022/feature/about/ExternalServices.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+import androidx.annotation.DrawableRes
+
+sealed interface ExternalServices {
+    val iconRes: Int
+    val packageName: String
+    val url: String
+    val contentDescription: String
+
+    data class Twitter(
```
Comment (takahirom) [inline feature-about/src/main/java/io/github/droidkaigi/confsched2022/feature/about/ExternalServices.kt:11]: Can we use object class or enum? I think we don't have to be able to be able to change this 👀

File: feature-about/src/main/java/io/github/droidkaigi/confsched2022/feature/about/About.kt [hunk 2]
```diff
@@ -26,7 +43,80 @@ fun About(
...
+    serviceType: ExternalServices,
+) {
+    try {
+        Intent(Intent.ACTION_VIEW).also {
+            it.setPackage(serviceType.packageName)
+            it.data = Uri.parse(serviceType.url)
+            context.startActivity(it)
+        }
+    } catch (e: ActivityNotFoundException) {
+        navigateToCustomTab(
```
Comment (takahirom) [inline feature-about/src/main/java/io/github/droidkaigi/confsched2022/feature/about/About.kt:106]: 👍

**Review by takahirom — APPROVED** [review]
Almost LGTM!

File: feature-about/src/main/java/io/github/droidkaigi/confsched2022/feature/about/ExternalServices.kt (hunk 1 above)
Comment (Corvus400) [reply feature-about/src/main/java/io/github/droidkaigi/confsched2022/feature/about/ExternalServices.kt]: @takahirom 
We have corrected it [here](https://github.com/DroidKaigi/conference-app-2022/pull/214/commits/41339549e0f1be9affe497371067fbd7ce27b52e).
It has already been approved, but please check again.🙇

**Review by takahirom — APPROVED** [review]
LGTM! Thanks for creating the foundation of DroidKaigi!

**Comment by takahirom** [comment]
Sorry, Please check AboutScreenTest 🙏

**Comment by Corvus400** [comment]
@takahirom 
The test was also modified.
Sorry for the repetition, but please check back.🙏

**Pull request merged**